app/uk/gov/hmrc/crdlrefdatadpsstub/config/Module.scala

```diff
@@ -17,11 +17,13 @@
 package uk.gov.hmrc.crdlrefdatadpsstub.config
 
 import com.google.inject.AbstractModule
+import uk.gov.hmrc.crdlrefdatadpsstub.service.{DefaultFileReader, FileReader}
 
 class Module extends AbstractModule {
 
   override def configure(): Unit = {
 
     bind(classOf[AppConfig]).asEagerSingleton()
+    bind(classOf[FileReader]).to(classOf[DefaultFileReader])
```

Comment:
One thing that you can do to avoid having to do this is to use an `@ImplementedBy` annotation on your implementation class, but both ways are totally fine